plugins/osdn/ovs/controller.go

```diff
@@ -63,11 +69,16 @@ func alreadySetUp(multitenant bool, localSubnetGatewayCIDR string) bool {
 	}
 	found = false
 	for _, flow := range flows {
-		if !strings.Contains(flow, "table=3") {
+		if !strings.Contains(flow, STAMP_TABLE) {
+			continue
+		}
+		idx := strings.Index(flow, STAMP_ACTION)
+		if idx < 0 {
 			continue
 		}
-		if (multitenant && strings.Contains(flow, "NXM_NX_TUN_ID")) ||
-			(!multitenant && strings.Contains(flow, "goto_table:9")) {
```

Comment:
Also supposed to check whether you updated openshift and restarted it, and we changed the flow rules in between.  I missed the plugin switch case, so I'll fix that up.